changelogs/fragments/4382-keycloak-add-missing-validate_certs-parameters.yml

```diff
@@ -0,0 +1,6 @@
+---
+bugfixes:
+  - keycloak_* - the documented `validate_certs` parameter was not taken into
+    account when calling the `open_url` function, thus enforcing certificate
+    validation even when `validate_certs` was set to `no`.
```

Comment:
```suggestion
  - keycloak_* - the documented ``validate_certs`` parameter was not taken into
    account when calling the ``open_url`` function in some cases, thus enforcing
    certificate validation even when ``validate_certs`` was set to ``false``.
```

Reply:
Thanks for the suggestion. It just has been applied.